clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/services/LaunchTemplateService.java

```diff
@@ -247,6 +247,13 @@ private RequestLaunchTemplateData buildLaunchTemplateData(
     networkInterfaceRequest.setDeviceIndex(0);
     networkInterfaceRequest.setGroups(description.getSecurityGroups());
 
+    // Nitro Enclave options
+    if (asgConfiguration.getEnableEnclave() != null) {
```

Comment:
this should be `description.getEnableEnclave()` instead of `asgConfiguration.getEnableEnclave()` since the launch template data being built here is for modifyLaunchTemplate operation. 
(FYI, this method shouldn't have a `AsgConfiguration` object - I will fix that in an upcoming PR)

And you also need to add this new field in [ModifyServerGroupLaunchTemplateDescription](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/deploy/description/ModifyServerGroupLaunchTemplateDescription.java#L20).